src/index.css

```
@@ -19,6 +19,12 @@ main {
   height: 100%;
   overflow-y: auto;
 }
+.plugins-table thead th,
+.scripts-table thead th {
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
```

When scrolling, the bottom border of the table heading is not visible.

```suggestion
}
.plugins-table thead th::before,
.scripts-table thead th::before  {
  content: '';
  width: 100%;
  position: absolute;
  bottom: 0;
  left: 0;
  border-bottom: #000 solid 1px;
}
```